### PR TITLE
Added practice task to bottom of intro to R (main.rmd/md/html files)

### PR DESCRIPTION
**Pull request by lwjohnst86**

### CODE REVIEW

File: lessons/introR/main.Rmd [hunk 1]
```diff
@@ -294,3 +294,11 @@ library(knitr)
 knit('main.Rmd') ## or can use knitr::knit('main.Rmd')
 
 ```
+
+# Challenge: Try this out for yourself! #
+
+Make a table with the means of Agriculture, Examination, Education,
+and Infant.Mortality by Fertility as a factor that is split in the
```
Comment (QuLogic) [inline lessons/introR/main.Rmd]: I'm just a bit confused by the "by Fertility". Do you mean "sorted by", "split by", "labelled by", etc.?

Comment (QuLogic) [inline lessons/introR/main.Rmd]: Or it just came to me, maybe you meant the _means_ averaged with respect to Fertility...

Comment (lwjohnst86) [inline lessons/introR/main.Rmd]: Yes, that's right. So the means of Agriculture for each category of Fertility that has been converted into a factor variable. How about:

> '..with the means of Agriculture... for each category of Fertility (hint: convert it into a factor by values >50 vs <50), and when Catholic is less than ...'

Does that make more sense?

Comment (QuLogic) [inline lessons/introR/main.Rmd]: Yea, sounds good.

**Pull request merged**